dist/api_data.py

```diff
@@ -1971,13 +1971,13 @@ def __ge__(self, other):
 'WT_CONNECTION.query_timestamp' : Method([
     Config('get', 'all_durable', r'''
         specify which timestamp to query: \c all_durable returns the largest timestamp such that
-        all timestamps up to that value have been made durable, \c last_checkpoint returns the
-        timestamp of the most recent stable checkpoint, \c oldest_timestamp returns the most
-        recent \c oldest_timestamp set with WT_CONNECTION::set_timestamp, \c oldest_reader
-        returns the minimum of the read timestamps of all active readers \c pinned returns
-        the minimum of the \c oldest_timestamp and the read timestamps of all active readers,
+        all timestamps up to that value have been made durable; \c last_checkpoint returns the
+        timestamp of the most recent stable checkpoint; \c oldest_timestamp returns the most
+        recent \c oldest_timestamp set with WT_CONNECTION::set_timestamp; \c oldest_reader
+        returns the minimum of the read timestamps of all active readers; \c pinned returns
+        the minimum of the \c oldest_timestamp and the read timestamps of all active readers;
         \c recovery returns the timestamp of the most recent stable checkpoint taken prior to a
-        shutdown and \c stable_timestamp returns the most recent \c stable_timestamp set with
+        shutdown, and \c stable_timestamp returns the most recent \c stable_timestamp set with
```

Comment:
Since the list's semicolon-delimited, this one should be a semicolon too.

Reply:
Fixed in 7c4e9a17e.